impeller/entity/geometry/geometry.h

```diff
@@ -48,6 +48,16 @@ ComputeUVGeometryCPU(
     Size texture_coverage,
     Matrix effect_transform);
 
+/// @brief Computes geometry and UV coordinates for a rectangle to be rendered.
+///
+/// UV is the horizontal and vertical axes within the texture.
+///
+/// @param source_rect      The rectangle to be rendered.
+/// @param texture_coverage The size of the texture to be rendered.
```

Comment:
A more accurate description for this would be something like: "The local space bounding box of the geometry."

Reply:
Done!